projects/ViTDet/vitdet/convModule_norm.py

```diff
@@ -0,0 +1,28 @@
+from mmcv.cnn import ConvModule
+
+class ConvModule_Norm(ConvModule):
+    def __init__(self, in_channels,
+                out_channels,
+                kernel, **kwargs):
+        super().__init__(in_channels, out_channels, kernel, **kwargs)
+
+        self.normType = kwargs.get('norm_cfg', {'type':''})
+        if self.normType is not None:
+            self.normType = self.normType['type']
+    
+    def forward(self, x, activate=True, norm=True):
+        for layer in self.order:
+            if layer == 'conv':
+                if self.with_explicit_padding:
+                    x = self.padding_layer(x)
+                x = self.conv(x)
+            elif layer == 'norm' and norm and self.with_norm:
+                if 'LN' in self.normType:
+                    x = x.permute(0, 2, 3, 1)
```

Comment:
We can implement a new LN layer that does the permute and normalization, then choose to use the new LN layer through norm_cfg. In this way, we can reduce most of the code that needs to use ConvModule.